app/messages/inbound/send-notification.js

```diff
@@ -36,10 +36,10 @@ const sendNotification = async (message) => {
 
     try {
       if (response) {
-        await logCreatedNotification(message, response.data.id)
+        await logCreatedNotification(message, emailAddress, response.data.id)
       } else {
-        await publishStatus(message, notifyStatus.INTERNAL_FAILURE, notifyError.response.data)
-        await logRejectedNotification(message, notifyError)
+        await publishStatus(message, emailAddress, notifyStatus.INTERNAL_FAILURE, notifyError.response.data)
```

Comment:
This is a bit of a long line now. Can we do anything to split this up a bit?

Reply:
Have pushed a change where I've reformatted the function call to wrap the arguments over multiple lines:
```
  await publishStatus(
      message,
      emailAddress,
      notifyStatus.INTERNAL_FAILURE,
      notifyError.response.data
  )
```
Is this sufficient? Otherwise was going to go with this option:
```
  const status = notifyStatus.INTERNAL_FAILURE
  const notifyErrorData = notifyError.response.data
  
  await publishStatus(message, emailAddress, status, notifyErrorData)
```
Let me know which is preferable. Thanks. 👍🏽

Reply:
I'd go with the second one